charts/nextcloud/templates/_helpers.tpl

```diff
@@ -47,3 +47,13 @@ Create chart name and version as used by the chart label.
 {{- define "nextcloud.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{- define "nextcloud.ingress.apiVersion" -}}
+{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- print "extensions/v1beta1" -}}
```

Comment:
nice, I like that you handled the check for previous versions

Reply:
I take no credit for the method, but I don't remember who I got the idea from